enterprise/server/test/integration/workflow/workflow_test.go

```diff
@@ -46,9 +52,39 @@ actions:
   - name: "Test action"
     triggers: { push: { branches: [ master ] } }
     bazel_commands: [ "build //:nop" ]
-`,
+` + osStr + archStr,
```

Comment:
nit: it is somewhat idiomatic to build the string inline, like this:

```suggestion
    os: "`+runtime.GOOS+`"
    arch: "`+runtime.GOARCH+`"
`
```

imo the whitespace is a bit clearer this way. (same comment for below)

Reply:
Oh nice couldn't figure out how to interpolate a variable